package.json

```diff
@@ -67,7 +67,7 @@
     "@polymer/paper-toast": "^3.0.1",
     "@polymer/paper-tooltip": "^3.0.1",
     "@polymer/polymer": "3.1.0",
-    "@thomasloven/round-slider": "0.3.7",
+    "@thomasloven/round-slider": "^0.4.1",
```

Comment:
Let's hope @thomasloven doesn't do breaking changes in minor releases 😄

Reply:
There's a reason it didn't have a ^ before... but I'll do my best.

Reply:
Well, fun fact: When having a 0 as major then each minor (or even patch) could contain breaking changes according to semver. See Point 4.  https://semver.org/ (and 6-8 all have this "x > 0" condition)  😉

Reply:
I will fix the ^ must have done that automatically when I upgraded